src/elements/Banner/Banner.tsx

```diff
@@ -34,6 +34,7 @@ const CloseButton = ({ onClick }) => {
 export interface BannerProps {
   dismissable: boolean
   message: string
+  backgroundColor?: string
```

Comment:
Would you mind updating [the test](https://github.com/artsy/palette/blob/7a01a2438206914bc8cd6fdedf6acaafd2074f28/src/elements/Banner/Banner.test.tsx) with this new API option?